starknet_devnet/starknet_wrapper.py

```diff
@@ -145,6 +142,10 @@ async def __get_state_root(self):
         state = await self.get_state()
         return state.state.shared_state.contract_states.root
 
+    def store_contract(self, address, contract, contract_class):
```

Comment:
Can you add type hints here? Also, can this method be applied in other places where we do ContractWrapper creation and self.contracts.store

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/203/commits/287f3e3d088c81e832767655c774ee45b303e298